.github/actions/create-jira-issues-for-community-activities/action.yml

```diff
@@ -0,0 +1,103 @@
+﻿name: Create Jira issues for community activities
+description: >
+  Creates Jira issues for community activities happening on GitHub, like issues, discussions, pull requests being
+  opened.
+
+inputs:
+  issue-component:
+    required: false
+    description: Optional component of the issues being created.
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Scripts
+      shell: pwsh
+      run: |
+        "${{ github.action_path }}" >> $Env:GITHUB_PATH
+        cd ${{ github.action_path }}
+        cd ../
+        "$((Get-Location).Path)/verify-submodule-pull-request" >> $Env:GITHUB_PATH
+
+    # Don't do anything for pull requests that are already related to an issue, i.e. their titles start with an issue
+    # key.
+    - name: Check for Jira Issue-Related Pull Request
+      id: issue-related-pr
+      shell: pwsh
+      run: |
+        Write-Output "::set-output name=is-not-issue-pr::$(-not(Check-PullRequestTitle ${{ github.event.pull_request.title }}))"
```

Comment:
This accidentally works, because we only need the first word anyway. But generally the `${{ ... }}` is preprocessor substitution, so you ought to wrap it in quotes.